public/app/ui/components/search/queryBuilder/query-builder.html

```diff
@@ -0,0 +1,32 @@
+<div class="query-builder" data-bind="with: root">
+    <div data-bind="template: $component.templateFor($data)"></div>
```

Comment:
The `template` binding is like a simplified `component` binding.  It doesn't create a view model, it only injects the content from a named template (the two `script` elements below) and binds to the current view model.  This line injects the `group-template` and it is bound to the root `Group` object (see `with: root` on line 1).